example/src/main/java/io/netty/example/http2/Http2ExampleUtil.java

```diff
@@ -24,5 +31,59 @@
      */
     public static final String UPGRADE_RESPONSE_HEADER = "Http-To-Http2-Upgrade";
 
+    /**
+     * Size for the array used to copy the input stream.s
+     */
+    private static final int BYTEARRAY_SIZE = 1024;
+
     private Http2ExampleUtil() { }
+
+    /**
+     * @param string the string to be converted to an integer.
+     * @param defaultValue the default value
+     * @return the integer value of a string or the default value, if the string is either null or empty.
+     */
+    public static int toInt(String string, int defaultValue) {
+        if (string != null && !string.isEmpty()) {
+            return Integer.valueOf(string);
+        }
+        return defaultValue;
+    }
+
+    /**
+     * Reads an InputStream into a byte array.
+     * @param input the InputStream.
+     * @return a byte array representation of the InputStream.
+     * @throws IOException if an I/O exception of some sort happens while reading the InputStream.
+     */
+    public static byte[] toByteArray(InputStream input) throws IOException {
+        byte[] buffer = new byte[BYTEARRAY_SIZE];
+        ByteArrayOutputStream output = null;
```

Comment:
nit: Consider using `ByteBuf` to do the heavy lifting for you. https://github.com/netty/netty/blob/master/buffer/src/main/java/io/netty/buffer/ByteBuf.java#L1514
and
https://github.com/netty/netty/blob/master/buffer/src/main/java/io/netty/buffer/ByteBuf.java#L1274